client/devdocs/design/index.jsx

```diff
@@ -50,14 +53,22 @@ import ExternalLink from 'components/external-link/docs/example';
 import FeatureGate from 'components/feature-example/docs/example';
 import FilePickers from 'components/file-picker/docs/example';
 import Collection from 'devdocs/design/search-collection';
+import fetchUsageStats from 'state/usage-stats/actions';
 
-export default React.createClass( {
+let DesignAssets = React.createClass( {
 	displayName: 'DesignAssets',
 
 	getInitialState() {
 		return { filter: '' };
 	},
 
+	componentWillMount() {
+		if ( config.isEnabled( 'devdocs/usage-stats' ) ) {
+			const { dispatchFetchUsageStats } = this.props;
+			dispatchFetchUsageStats();
```

Comment:
This is what I am already doing I think.
I was talking about fetching the stats again if there is a re-rendering and the current stats (in memory) are different from the ones from a new deploy.

In details:
* The user visits the URL and gets some stats
* A new version of Calypso is deployed and the stats are now inconsistent
* There is a re-rendering of the page without a refresh
* The user gets the old stats